_posts/2020-03-05-git-secret.md

```diff
@@ -76,11 +77,11 @@ To import the public key of someone else (to share the secret with them for inst
 gpg --import public-key.gpg
 ```
 
-Be sure to use a secure channel to share your public key!
+To make sure you get the original public keys of the indicated persons, be sure to use a secure channel to transfer it, or use a service you trust like Keybase to retrieve their public key. Otherwise you could grant the wrong person access to your secrets by mistake!
```

Comment:
please change to  

"or use a service you trust, preferably one that uses encryption such as Keybase, to retrieve their public key".